.github/workflows/ci.yml

```diff
@@ -6,19 +6,18 @@ on:
       - 'main'
 
 env:
-  BUILDER_VERSION: v0.9.62
+  BUILDER_VERSION: v0.9.72  
   BUILDER_HOST: https://d19elf31gohf1l.cloudfront.net
   BUILDER_SOURCE: releases
   PACKAGE_NAME: aws-c-common
   LINUX_BASE_IMAGE: ubuntu-18-x64
   RUN: ${{ github.run_id }}-${{ github.run_number }}
-  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-  AWS_REGION: us-east-1
+  CRT_CI_ROLE: ${{ secrets.CRT_CI_ROLE_ARN }}
+  AWS_DEFAULT_REGION: us-east-1
 
 jobs:
   linux-compat:
-    runs-on: ubuntu-20.04 # latest
+    runs-on: ubuntu-22.04 # latest
```

Comment:
Why not 24?

Reply:
some of the older compilers are not available on 24 (clang 3 and gcc 4.8 if i remember correctly). dropping those from our ci is a bridge we can cross later

Reply:
BTW, Does this version of Ubuntu matter since our Docker base image is Ubuntu 18?

Reply:
nope: should not matter too much (i think our 18 ubuntu image might be actually 20, but i cant find actual docker file)
This is just a prophylactic image bump.

Reply:
actually you are right. it should not affect the compiler cause underlying image is old. let me retest. i saw old compilers failing when i moved to 24, so i assumed its related, but maybe it is not.